test/multiapi/packages/multiapi-test-2017-10-01/rollup.config.js

```diff
@@ -0,0 +1,31 @@
+import nodeResolve from "rollup-plugin-node-resolve";
```

Comment:
@bterlson this is an example of one of our generated rollup configs

Reply:
Looks very nice! But do you need the commonjs plugin?